src/main/java/com/booking/config/DatabaseConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.booking.superride.config;
+
+import org.hibernate.internal.TransactionManagement;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.support.TransactionTemplate;
+
+@Configuration

```

Comment:
We dont need proxy class created for this so change this line to `@Configuation(proxyBeans =false)`